src/types/Todo.ts

```diff
@@ -0,0 +1,9 @@
+import { User } from './User';
+
+export interface Todo {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number,
+  user: User | undefined,
```

Comment:
Shorter syntax for the same thing
```suggestion
  user?: User,
```